packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.js

```diff
@@ -103,13 +120,19 @@ const BpkGraphicPromo = (props: Props) => {
 
   const getTextClasses = (baseClass: string) =>
     getClassName(baseClass, `${baseClass}--${textAlign}`);
+
   return (
-    <BpkCard
```

Comment:
In thinking about this have you tried making the BpkCard `non-atomic` https://backpack.github.io/components/card?platform=web#non-atomic as this might be the solution that you need for this and not need too much of the work you are doing?